app/controllers/auth/two_factor_authentication_controller.rb

```diff
@@ -22,4 +22,12 @@ def after_two_factor_fail_for(resource)
       render :show, status: :unprocessable_entity
     end
   end
+
+  def after_two_factor_success_path_for(resource)
+    if resource.is_a?(User) && resource.support?
+      "/organisations"
+    else
+      super
+    end
```

Comment:
Had to do it this way to prevent it blowing up with the current admin user code. This can be tidied up if we decide to remove the admin user and active admin